### ISSUE TITLE
API Documentation needs explanations

### ISSUE THREAD

**Issue by domoritz** [cirospaciari/socketify.py#158]
The API docs at https://docs.socketify.dev/api.html don't describe the parameters or what the methods do.

**Comment by cirospaciari** [comment]
Yeah and we need a better site too, this is too little information.
We should be able to search for something and get the API with description and at least one use case of example.

**Comment by chask-stealth-mode** [comment]
If it help, I've created a GPT with the documentation. Might be useful here. 

https://github.com/cirospaciari/socketify.py/issues/162#issue-2116693738

**Comment by greg-martinez44** [comment]
I can help with docs if this is still available.

**Comment by avylera** [comment]
Hello, I loved to be assigned to this task!

**Comment by cirospaciari** [comment]
Every one can help :D just open a PR and I will take a look to merge it

**Comment by Ashleycodesseo** [comment]
@cirospaciari I looked at the website. Do you still need help with documentation? Is this a REST API? I'm looking to gain some experience with REST API documentation and I'd be happy to help

**Comment by cirospaciari** [comment]
Would love help with this!

**Comment by Ashleycodesseo** [comment]
@cirospaciari great! so what do you need me to do? From what I'm seeing you need parameters and methods described for each function? Let me know what you need and I will get to it

**Comment by cirospaciari** [comment]
@Ashleycodesseo basically something similar to nodejs docs https://nodejs.org/api/http2.html

Methods, and types described, options described and some code examples.

**Comment by Ashleycodesseo** [comment]
hey @cirospaciari just wanted to update you. I'm still working on the documentation. Will be working on it over the next few days

**Comment by Ashleycodesseo** [comment]
Ok so @cirospaciari I forked my own copy of the code and I'm looking thru it but I'm so confused. (*forgive me I'm not much of a coder so I'm learning bit by bit as I go) but I'm trying to understand the functions you want me to document as listed on https://docs.socketify.dev/api.html . But How can I test them? Wouldn't a dev implement them in their own app? Should I append these functions to my own copy to see how they work?

**Comment by Ashleycodesseo** [comment]
nevermind @cirospaciari I actually figured it out using docker!

**Comment by Ashleycodesseo** [comment]
@cirospaciari actually no I didn't lol. I ran the app locally on my computer and it just showed the "Hello World from Socketify" What am I doing wrong?

**Comment by cirospaciari** [comment]
@Ashleycodesseo I'm a little confuse with your question, if you are running the Hello World example the application will just show this message. socketify is a HTTP and WebSocket server you can write web application using it as base, it's don't provide any front-end and is really raw, the goal is high performance Web APIs and WebSocket communication.
